libs/blocks/card/card.js

```diff
@@ -57,8 +57,6 @@ const init = (el) => {
   const base = miloLibs || codeRoot;
   loadStyle(`${base}/deps/caas.css`);
 
-  const section = el.closest('.section');
-  section.classList.add('milo-card-section');
```

Comment:
way the class `milo-card-section` needs to be removed?

Reply:
@VKniaz and @Axelcureno might be able to clarify, I just removed code duplication, not sure what is it needed for.

Reply:
milo-card-section class is used by consonant styles and should not be removed. I see @3ch023 is just moving this changes to addWrapper in cardUtils.js and updating the usage in card.js so I'm good with this 👍🏼